app/src/main/java/com/fivesysdev/standwithukraine/ui/StatisticFragment.kt

```diff
@@ -61,11 +70,14 @@ class StatisticFragment : Fragment(R.layout.fragment_statistic) {
 
     private fun setDayStatistic(dayStatistic: DayStatistic?) {
         checkBlockNextButton()
-        if (dayStatistic != null) {
-            binding.recyclerview.adapter = StatisticAdapter(dayStatistic.statistic)
-        }
+        updateDataList(dayStatistic)
         binding.tvDate.text = viewModel.date.toString()
-        setEmptyDataObserver()
+    }
+
+    private fun updateDataList(dayStatistic: DayStatistic?) {
+        data.clear()
+        data.addAll(dayStatistic?.getStatisticsPair()?:listOf())
```

Comment:
please, use code formatting tool
To automatically format your code in the current source code window, use Cmd+Option+L (on Mac) or Ctrl+Alt+L (on Windows and Linux).